db/seeds.rb

```diff
@@ -41,14 +41,20 @@
   'Townhouse Apts'       => 30
 }
 
+directions = ['North', 'South']
+
 stops.each do |route_number, stop_names|
-  stop_names.each do |stop_name|
+  stop_names.each_with_index do |stop_name, sequence|
     # Anytime in the last two months
     Timecop.freeze rand(86_400).minutes.ago do
+      # don't want the sequences to start with 0.
+      sequence = sequence + 1
       stop = BusStop.find_or_initialize_by name: stop_name
       stop.hastus_id = hastus_ids.fetch(stop_name)
-      stop.routes << routes.fetch(route_number)
       stop.save!
+      route = routes.fetch(route_number)
+      direction = directions.sample
```

Comment:
That doesn't sound right. This would be a random smattering of integers between north and south.

We want north: 1,2,3,4, etc. and south: 1,2,3,4, etc.

Reply:
?????

Reply:
There should be a contiguous set of sequence numbers per direction, starting at 1. My reading is that this will create for example

Sequence | Direction
:--- | ---
1 | North
2 | South
3 | South
4 | North
5 | North
6 | South
7 | North

etc.